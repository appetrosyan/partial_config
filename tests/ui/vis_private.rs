#[derive(partial_config::HasPartial)]
struct ConfigurationError {

}

fn main() {}
