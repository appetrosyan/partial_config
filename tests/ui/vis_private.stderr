error: Cannot implement `HasPartial` for a private structure.

         = help: If your structure is private, it is better to convert to it with an `Into::into` rather than directly derive `HasPartial`, which by definition will expose some of the fields

 --> tests/ui/vis_private.rs:1:10
  |
1 | #[derive(partial_config::HasPartial)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `partial_config::HasPartial` (in Nightly builds, run with -Z macro-backtrace for more info)
